### PR TITLE
Add staging project for cloud provider for kind

### PR DESCRIPTION
**Pull request by ameukam**

### CODE REVIEW

**Comment by ameukam** [comment]
cc @aojea @BenTheElder

**Comment by aojea** [comment]
the ci complains about the name?

**Comment by ameukam** [comment]
more specifically length of the project id. we need to drop at least 1 character.

File: groups/sig-testing/groups.yaml [hunk 1]
```diff
@@ -282,3 +282,13 @@ groups:
     members:
     - eddiezane@gmail.com
     - k8s-infra-prow-oncall@kubernetes.io
+
+  - email-id: k8s-infra-staging-cloud-provider-kind@kubernetes.io
+    name: k8s-infra-staging-cloud-provider-kind
```
Comment (aojea) [inline groups/sig-testing/groups.yaml:287]: and if we drpo the `infra` ??

Comment (ameukam) [reply groups/sig-testing/groups.yaml:287]: The tooling we are using to create those container repos requires we put `k8s-infra-staging` as prefix.

We can use the existing `k8s-staging-kind` to push the images. Are intending to do promotion of the images built for this project?

Comment (aojea) [reply groups/sig-testing/groups.yaml:287]: I'm find using the kind bucket if there is no more concerns

> Are intending to do promotion of the images built for this project?

AFAIK the official images are pushed directly to dockerhub

Comment (ameukam) [reply groups/sig-testing/groups.yaml:287]: @BenTheElder Ok we use `k8s-staging-build` to host images for the cloud provider ?

Comment (BenTheElder) [reply groups/sig-testing/groups.yaml:287]: we have an existing project for kind, while we wanted two source repos for go module reasons I don't mind if both source repos share the staging project.

**Pull request closed without merging**